app/views/signs/search.html.haml

```diff
@@ -26,13 +26,13 @@
                     .maori_gloss.small-12.text-left= sign.gloss_maori
                   - if sign.word_classes.present?
                     .word_gloss.small-12.text-left= sign.word_classes
-                .link_and_drawing_container.small-12
-                  .link_container.float-left
-                    = link_to "learn more", sign_url(sign.id), {class: "show_link float-left"}
-                    %span.unicode_arrow.float-left &rarr;
-                  .drawing_container.float-left
-                    = link_to sign_url(sign.id), {:class => 'drawing grid_drawing js-ga-link-submission', onclick: "_gaq.push(['_trackEvent', 'Search', 'Click', '#{query_text}-#{sign.id}', #{@signs.find_index(sign)  + 1}]);"} do
-                      = image_tag "signs/150-100/#{sign.drawing}"
+          .link_and_drawing_container.small-12
+            .link_container.float-left
+              = link_to "learn more", sign_url(sign.id), {class: "show_link float-left"}
+              %span.unicode_arrow.float-left &rarr;
+            .drawing_container.float-left
+              = link_to sign_url(sign.id), {:class => 'drawing grid_drawing js-ga-link-submission', onclick: "_gaq.push(['_trackEvent', 'Search', 'Click', '#{query_text}-#{sign.id}', #{@signs.find_index(sign)  + 1}]);"} do
```

Comment:
I don't think it needs to be a hash at all, it can just be
` +              = link_to sign_url(sign.id), class:  'drawing grid_drawing js-ga-link-submission', onclick: "_gaq.push(['_trackEvent', 'Search', 'Click', '#{query_text}-#{sign.id}', #{@signs.find_index(sign)  + 1}]);" do `

some of the complexity of this line can be simplified in a helper.
:elephant:  because it was already a mess before.